Makefile

```diff
@@ -1,9 +1,12 @@
 RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
-BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
+# allow for passing buildtags to make or setting the default
+DEFAULT_BUILDTAGS=seccomp
+BUILDTAGS := $(if $(BUILDTAGS),$(BUILDTAGS),$(DEFAULT_BUILDTAGS))
```

Comment:
Uhh, this should already work without doing this -- Make lets you override variables on the command line directly like `make BUILDTAGS='a b c' ...`; is it not working?